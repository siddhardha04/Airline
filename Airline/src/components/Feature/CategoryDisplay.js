import React from "react";

export default function CategoryDisplay() {
  return (
    <div className="categories">
      <div className="category-data">
        <div className="category-circle green"></div>
        <div>:Not checked-in</div>
      </div>
      <div className="category-data">
        <div className="category-circle red"></div>
        <div>:checked-in</div>
      </div>
      <div className="category-data">
        <div className="category-circle yellow"></div>
        <div>:Infant</div>
      </div>
      <div className="category-data">
        <div className="category-circle blue"></div>
        <div>:Handicap</div>
      </div>
    </div>
  );
}
