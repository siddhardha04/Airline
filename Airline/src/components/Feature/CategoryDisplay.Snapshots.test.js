import React from "react";
import CategoryDisplay from "./CategoryDisplay";
import renderer from "react-test-renderer";

it("Will display seat categories", () => {
  const tree = renderer.create(<CategoryDisplay />);
  expect(tree).toMatchSnapshot();
});
