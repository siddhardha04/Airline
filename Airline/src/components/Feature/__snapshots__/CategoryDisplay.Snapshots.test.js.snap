// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Will display seat categories 1`] = `
<div
  className="categories"
>
  <div
    className="category-data"
  >
    <div
      className="category-circle green"
    />
    <div>
      :Not checked-in
    </div>
  </div>
  <div
    className="category-data"
  >
    <div
      className="category-circle red"
    />
    <div>
      :checked-in
    </div>
  </div>
  <div
    className="category-data"
  >
    <div
      className="category-circle yellow"
    />
    <div>
      :Infant
    </div>
  </div>
  <div
    className="category-data"
  >
    <div
      className="category-circle blue"
    />
    <div>
      :Handicap
    </div>
  </div>
</div>
`;
